pkgs/top-level/all-packages.nix

```diff
@@ -10444,6 +10444,8 @@ in
   };
   purePackages = recurseIntoAttrs (callPackage ./pure-packages.nix {});
 
+  pycalver = callPackage ../applications/version-management/pycalver { };
```

Comment:
This position is wrong. Around line 6667 should be right.

Reply:
Thanks, was not sure about position.